package.json

```diff
@@ -15,7 +15,8 @@
     "build": "babel src -d dist",
     "build:watch": "babel src -d dist --watch",
     "flow": "flow",
-    "jest": "jest"
+    "jest": "jest",
+    "preinstall": "node ./scripts/check-version.js"
```

Comment:
Running this preinstall breaks on a fresh install. `semver` is a dependency and we can't use dependencies in our preinstall check since they haven't been installed yet. So you end up with this:

```
$ npm i

> @automattic/vip@1.0.0-dev-build9 preinstall /Users/mjangda/Sites/vipgo/public/cli
> node ./scripts/check-version.js

module.js:540
    throw err;
    ^

Error: Cannot find module 'semver'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/mjangda/Sites/vipgo/public/cli/scripts/check-version.js:1:78)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @automattic/vip@1.0.0-dev-build9 preinstall: `node ./scripts/check-version.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @automattic/vip@1.0.0-dev-build9 preinstall script.
```